lib/timeline/Core.js

```diff
@@ -245,6 +245,10 @@ Core.prototype._create = function (container) {
   }
 
   function handleDrop(event) {
+    // FIX drop redirect in FIREFOX : START (issue #2842)
```

Comment:
Please remove comments. You can leave one comment of "prevent redirect to blank page - Firefox"